### PR TITLE
Default min number of ECS Service container tasks to 1 for developers

### PR DESCRIPTION
**Pull request by dwyeradam**
To override for developer environments the defaults defined by the following PRs:
https://github.com/ONSdigital/eq-ecs-deploy/pull/8
https://github.com/ONSdigital/eq-author-deploy/pull/10
https://github.com/ONSdigital/eq-survey-runner-deploy/pull/17

### CODE REVIEW

**Comment by dwyeradam** [comment]
@jonnyshaw89 is there a reason we're currently tied to v1.0.0 for eq-ecs-deploy?

**Comment by jonnyshaw89** [comment]
@dwyeradam just so it’s tied to a release version. You can create a new release version and point to that

File: developer_defaults.tf [hunk 1]
```diff
@@ -290,6 +295,11 @@ variable "survey_launcher_jwt_signing_key_path" {
   default     = "jwt-test-keys/sdc-user-authentication-signing-rrm-private-key.pem"
 }
 
+variable "application_min_tasks" {
```
Comment (jonnyshaw89) [inline developer_defaults.tf]: We shouldn't reuse this variable, we should have one for each service.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Review by MIslam92 — APPROVED** [review]
'Squishy Squashy'

**Pull request merged**